widget/.parcelrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
(This file is JSON5](https://parceljs.org/features/plugins/#.parcelrc), so let's use the same 2-space indent as other files.

Go ahead and add this file name to `.editorconfig` too.